### ISSUE TITLE
Docker drop in service for flannel setup incorrect

### ISSUE THREAD

**Issue by urkle** [projectatomic/atomic-site#349]
The documentation on http://www.projectatomic.io/docs/gettingstarted/ mentioned to add a "drop-in" for systemd to override docker.. However the example file is incorrect and yields the following error when startup up docker.
```
Aug 07 23:05:45 atomic01.kaos.realm systemd[1]: docker.service has more than one ExecStart= setting, which is only allowed for Type=oneshot services. Refusing.
```
There was an open bug up on the docker github that references how to fix this issue.

https://github.com/docker/docker/issues/14491

The solution is simply to ADD an 
```
ExecStart=
```
before the full ExecStart= line.   I've tested this locally and it does indeed work as expected.

**Comment by urkle** [comment]
Actually this "drop-in" is no longer needed in current CentOS7 atomic host.   the stock setup auto picks up flannel setup.

Also, the documentation needs to mention that the edits /etc/sysconf/flanneld needs to happen on the atomic-master as well!

**Comment by jberkus** [comment]
I'm a little unclear on all of the corrections needed?  Can you break them out?

**Comment by urkle** [comment]
@jberkus yeah.. sure.  I was reporting these as I was figuring out how to get an atomic setup in VMs and after getting past one hurdle hit another one, but now it's all working.

First this is based on using the latest CentOS 7 Atomic Host.
- [ ] the "drop-in" flanneld service is not needed (that is the /etc/systemd/system/docker.service.d/10-flanneld-network.conf file).   Just simply configuring the /etc/sysconfig/flanneld and renabling the flanneld service is enough to have docker service pickup the flannel configuration.
  - this obviates my original note about ExecStart=
- [ ] the documentation needs to mention in the section about editing /etc/sysconfig/flanneld to ALSO change that file and enable the service on the master!

**Comment by nzwulfin** [comment]
All of this points to the fact that the doc hasn't been retooled in a while and was based on Fedora 22 Atomic Host.

Since there may be material differences between the CentOS and Fedora Atomic Hosts at any point in time, should look to split the doc or have call outs for differences?

**Comment by urkle** [comment]
I can try and go through the doc on a F24 later this week and see how it differs from CentOS7..  If there are minimal (or no differences) then the call-outs seems the better option.. If there are significant differences then the split approach would be better.
